sirepo/job_supervisor.py

```diff
@@ -883,7 +883,10 @@ def res(**kwargs):
         ):
             return PKDict(state=job.MISSING, reason='computeJobSerial-mismatch')
         if self.db.isParallel or self.db.status != job.COMPLETED:
-            return res(state=self.db.status)
+            return res(
+                state=self.db.status,
+                dbUpdateTime=_strf_unix_time(self.db.dbUpdateTime),
```

Comment:
Seems to me this is presentation logic. Why do we need to send back a formatted time stamp?

Reply:
From my experience, JS has [weak capabilities for formatting timestamps](https://css-tricks.com/everything-you-need-to-know-about-date-in-javascript/). We are already doing it on the server so I just continued down that path. Thinking about it more it looks like [angularjs provides what we need](https://docs.angularjs.org/api/ng/filter/date). Should I use that instead?

Reply:
Using a date formatter is fine. I would ideally like to encapsulate this so we don't become more dependent on angular. We would want all dates formatted the same way so make a routine (e.g. appState.formatDate) and put all the code there.

Reply:
https://github.com/radiasoft/sirepo/pull/2690/commits/baaf84a3bdfc7d0bba4ec6793c0eb670645b610f

Should I fix the [jobs page](https://github.com/radiasoft/sirepo/blob/baaf84a3bdfc7d0bba4ec6793c0eb670645b610f/sirepo/job_supervisor.py#L511) too?

Reply:
Yes, please

Reply:
https://github.com/radiasoft/sirepo/pull/2690/commits/4d21167b3a5bfc67863e22bdd19621a3288bbb87